src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -24,12 +24,30 @@ import { BoldText, RegularText } from '../Text';
 import { Touchable } from '../Touchable';
 import { Wrapper } from '../Wrapper';
 
+const { IMAGE_TYPE_REGEX, PDF_TYPE_REGEX, URL_REGEX } = consts;
+
+const deleteCalendarAlert = (onPress: () => Promise<void>) =>
+  Alert.alert('Hinweis', 'Möchten Sie das Event löschen?', [
+    { text: 'Abbrechen', style: 'cancel' },
+    { text: 'Löschen', onPress, style: 'destructive' }
+  ]);
+
 // eslint-disable-next-line complexity
 export const VolunteerFormCalendar = ({
   navigation,
+  route,
   scrollToTop,
   groupId
 }: StackScreenProps<any> & { scrollToTop: () => void; groupId?: number }) => {
+  const calendarData = route.params?.calendarData ?? undefined;
```

Comment:
`route.params?.calendarData` would be enough, that returns `undefined` if not defined

Reply:
01b627a